### PR TITLE
Provide keras normalization layer in tf

### PR DESCRIPTION
**Pull request by alekstheod**
Running benchmarks which are based on the old implementation of the tensorflow (1.x) or a compatibility layer was not possible without this change. Batch normalization layer was removed from the latest version of the framework. Instead we use the Batch normalization from the keras api.

### CODE REVIEW

**Comment by jayfurmanek** [comment]
I think probably the best solution is to update the user code. We could try pushing this upstream though I suspect it wouldn't be accepted.

**Comment by alekstheod** [comment]
> I think probably the best solution is to update the user code. We could try pushing this upstream though I suspect it wouldn't be accepted.

I am not sure what is the correct fix here. From our perspective the compatibility later will not be working without that change. This change was done just to run our old benchmark project with the newest tf. I will discuss that in our daily and check how to handle that.

**Comment by i-chaochen** [comment]
> This change was done just to run our old benchmark project with the newest tf.

our old benchmark project? do you mean DLM or something else?

**Comment by alekstheod** [comment]
> > This change was done just to run our old benchmark project with the newest tf.
> 
> our old benchmark project? do you mean DLM or something else?

I mean this project: https://github.com/ROCm/benchmarks
